layouts/partials/sections/event-highlights.html

```diff
@@ -15,7 +15,7 @@ <h2 class="heading heading--2">
                     {{ $input.subtitle }}
                 </h2>
 
-                <p class="paragraph">{{ $input.description | markdownify }}</p>
+                <p class="paragraph event-highlights-section__description--paragraph">{{ $input.description | markdownify }}</p>
```

Comment:
I would consider `--paragraph` more an element than a modifier. What about `--dimmed`? Maybe even globally for paragraph (`paragraph paragraph--dimmed`).

Reply:
fixed in the next comment